package.json

```diff
@@ -26,40 +26,44 @@
   "license": "Apache-2.0",
   "repository": {
     "type": "git",
-    "url": "https://github.com/gxa/my-package.git"
+    "url": "https://github.com/ebi-gene-expression-group/scxa-home-landing-page"
   },
   "dependencies": {
-    "atlas-homepage-cards": "file:../atlas-homepage-cards",
-    "file-loader": "^2.0.0",
+    "atlas-homepage-cards": "^1.2.0",
+    "atlas-latest-experiments-cards": "^1.2.1",
     "prop-types": "^15.6.2",
     "react": "^16.4.1",
     "react-dom": "^16.4.1",
     "react-router": "^4.3.1",
     "react-router-dom": "^4.3.1",
-    "urijs": "^1.19.1",
-    "url-loader": "^1.1.2"
+    "styled-components": "^4.1.2",
+    "urijs": "^1.19.1"
   },
   "devDependencies": {
-    "ajv": "^6.5.2",
-    "babel-cli": "^6.26.0",
-    "babel-core": "^6.26.3",
-    "babel-loader": "^7.1.5",
-    "babel-plugin-transform-object-rest-spread": "^6.26.0",
-    "babel-polyfill": "^6.26.0",
-    "babel-preset-env": "^1.7.0",
-    "babel-preset-react": "^6.24.1",
+    "@babel/cli": "^7.1.2",
+    "@babel/core": "^7.1.2",
+    "@babel/polyfill": "^7.0.0",
+    "@babel/preset-env": "^7.1.0",
+    "@babel/preset-react": "^7.0.0",
+    "ajv": "^6.5.4",
+    "babel-core": "^7.0.0-bridge.0",
+    "babel-jest": "^23.6.0",
+    "babel-loader": "^8.0.4",
     "clean-webpack-plugin": "^0.1.19",
-    "css-loader": "^1.0.0",
-    "enzyme": "^3.3.0",
-    "enzyme-adapter-react-16": "^1.1.1",
-    "fetch-mock": "^7.2.0",
-    "jest": "^23.4.0",
-    "node-fetch": "^2.2.0",
-    "react-test-renderer": "^16.4.1",
+    "coveralls": "^3.0.2",
+    "css-loader": "^1.0.1",
+    "enzyme": "^3.7.0",
+    "enzyme-adapter-react-16": "^1.6.0",
+    "eslint": "^5.8.0",
+    "eslint-config-gene-expression": "^0.4.2",
+    "fetch-mock": "^7.2.2",
+    "jest": "^23.6.0",
+    "react-test-renderer": "^16.6.0",
     "style-loader": "^0.23.1",
-    "webpack": "^4.16.0",
-    "webpack-command": "^0.4.1",
-    "webpack-serve": "^2.0.2",
+    "url-loader": "^1.1.2",
+    "webpack": "^4.23.1",
+    "webpack-cli": "^3.1.2",
+    "webpack-dev-server": "^3.1.10",
     "whatwg-fetch": "^3.0.0"
   }
```

Comment:
I can see a few dev dependencies which aren’t used. I leave you the exercise of finding and removing them. ;)